backend/src/routes/account/store/account_schemas.py

```diff
@@ -0,0 +1,19 @@
+from pydantic import BaseModel, EmailStr, constr
+from typing import Optional
+from bson import ObjectId
+
+class UserCreate(BaseModel):
+    email: EmailStr
+    password: constr(min_length=8)
+    full_name: str
+
+class UserResponse(BaseModel):
+    id: Optional[str]
+    email: EmailStr
+    full_name: str
```

Comment:
The validation here is of type 
email and full_name but if you see in `backend/src/routes/account/store/account_models.py` then the model there is of type
```
  account_id: int
    username: str=Indexed(unique=True)
    password: str
```

This shown that there is no consistency between sharing of models. How to fix?
For example
1. I am storing username and password in the database, I will put validations on the username and password instead of email and password. 
2. ALSO, We need to store email, phoneNumber and password so update the account_models too